src/views/Picklists.vue

```diff
@@ -12,6 +12,16 @@
     </ion-header>
     
     <ion-content :fullscreen="true">
+      <ion-list v-if="completedPicklist.length">
+        <ion-list-header lines="none">
+          <ion-label>{{ $t("Completed") }}</ion-label>
+        </ion-list-header>
+        <!-- TODO : We are only storing the id of completed picklist, so not showing any other information right now -->
+        <ion-item v-for="picklist in completedPicklist" :key="picklist.picklistId">
```

Comment:
I think in completedPicklist, we are directly storing the ids, so the `:key` becomes invalid here, WDYT?

Reply:
Yes Sir, We can also change it with `:key="picklist"` .